### ISSUE TITLE
Consider publishing snapshots

### ISSUE THREAD

**Issue by ZacSweers** [tschuchortdev/kotlin-compile-testing#68]
With KSP being an actively developed project, it's not uncommon for changes in this project to be out of sync and thus not usable for others. For instance, I'm unable to use this again until a new release is cut due to the recent package renames in KSP, even though it's been merged here already. Would you consider publishing snapshots for those that want to try out earlier versions?

**Comment by ZacSweers** [comment]
Sidenote - these are also very easy to maintain and automate with GH actions if you ever reconsider #25

**Comment by ZacSweers** [comment]
Note that jitpack isn't a reliable option, as the last two releases both failed https://jitpack.io/com/github/tschuchortdev/kotlin-compile-testing/1.2.11/build.log

**Comment by evant** [comment]
For what it's worth, the jitpack error looks like the same one I was seeing locally, so it might be worth fixing that regardless.

**Comment by tschuchortdev** [comment]
I will publish a new version for you later today. As for completely automated releases: I'm working on it but I have a lot of other things on my plate right now. In the past few weeks I have already migrated the Gradle plugin to make automated releasing with Gradle possible, so I no longer have to do it manually on the Sonatype Nexus website (at least most of the time. The Gradle plugin is pretty unreliable). The secret management and CI integration is still missing. My domain is Gitlab CI so I'll have to read up on how to do that with Github Actions.  

@evant I have fixed the jitpack error that you reported a while ago but it was not enough. There still seem to be other problems with disabling the signing when no credentials are present (jitpack artifacts aren't signed) and moving the build artifacts in the right location where jitpack can find them. I also want to automate the creation of releases on Github with Github Actions so jitpack can pick them up.
